examples/monolithic_build_multilevel_native/Makefile

```diff
@@ -125,16 +120,11 @@ $(BUILD_DIR)/%.S.o: %.S
 	$(Q)[ -d $(@D) ] || mkdir -p $(@D)
 	$(Q)$(CC) -c $(CFLAGS) $(ASMFLAGS) $(INC) $^ -o $@
 
-$(LIB_NAME_FULL): $(MLK_OBJ_C) $(MLK_OBJ_ASM)
-	$(Q)echo "AR  $@"
-	$(Q)[ -d $(@D) ] || mkdir -p $(@D)
-	$(Q)$(AR) rcs $@ $^
-	$(Q)strip -S $@
-
-$(BINARY_NAME_FULL): $(APP_SOURCE) $(LIB_NAME_FULL)
+$(BINARY_NAME_FULL): $(APP_SOURCE) $(MLK_OBJ_ASM)
 	$(Q)echo "CC  $@"
 	$(Q)[ -d $(@D) ] || mkdir -p $(@D)
 	$(Q)$(CC) $(CFLAGS) $(INC) $^ -o $@
+	$(Q)strip -S $@
```

Comment:
Should this include the cross prefix?

Reply:
Good question. If it is not necessary -- which appears to be the case -- I'd prefer to leave it as is.